test/blocks/editorial-card/mocks/body.html

```diff
@@ -118,7 +118,7 @@ <h3 id="open-key-variant-per-breakpoint-media-has-footer-row">Open key-variant,
     </div>
   </div>
   <h2>extra static row</h2>
-    <div class="editorial-card click">
+    <div class="editorial-card click empty-footer">
```

Comment:
Isn't this supposed to be just `empty` now? 🤔

Reply:
This isn't needed at all since it's not an authored variant. It was more a label for dev purposes and has been moved into the heading copy instead.